### DISCUSSION TITLE
Conjunto Prismático

### DISCUSSION THREAD

**Discussion by ghost** [leogianfagna/IssueTracker#985] [Ideia]
### Servidor afetado

Henesys

### Qual o propósito da ideia?

Um conjunto que serve para divulgar/tornar a skin acessível sem perder a exclusividade

### Descrição da ideia

**O Conjunto Prismático** é um conjunto especial que se transforma para copiar as skins da última caixa adicionada ao jogo. Isso significa que as skins estão sempre mudando conforme novas caixas chegam.

Como funciona:
Quando uma nova caixa de skins é lançada (por exemplo, a caixa "Calamidade"), o Conjunto Prismático copia automaticamente as skins dessa caixa.
Depois, se outra caixa for adicionada ou retornar (como a caixa "Oriental"), o Conjunto Prismático muda novamente para imitar as skins dessa nova caixa.

Vantagem:
- Isso cria um sistema dinâmico onde as skins do Conjunto Prismático nunca ficam ultrapassadas, sempre acompanhando as novidades do jogo.
- Cria divulgação da caixa atual, pois o número de pessoas usando vai aumentar, então vai ser comum ver alguém com ela, gerando interesse.
- Se o jogador gostar muito de uma skin específica que o Conjunto Prismático imitou, ele terá que comprá-la diretamente da caixa original para adquirir essa versão permanentemente. 

### Aplicação

O Conjunto Prismático poderia ser adquirido através do sistema de prestígio ou em uma caixa com preço reduzido.

**Comment by leogianfagna** [comment]
bem interessante a ideia mas realmente é difícil medir o impacto de aplicação dela no servidor.

**Reply by ghost** [reply]
Entendi, pensei que a dificuldade em obter acabaria compensando. De qualquer forma, valeu pelo feedback na ideia!
